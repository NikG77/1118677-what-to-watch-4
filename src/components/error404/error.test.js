import React from "react";
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import Error404 from "./error404.jsx";
import history from "../../history.js";


it(`Error404 component render correctly`, () => {

  const tree = renderer
    .create(
        <Router history={history}>
          <Error404 />
        </Router>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
