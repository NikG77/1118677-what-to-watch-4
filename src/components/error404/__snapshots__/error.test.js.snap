// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 component render correctly 1`] = `
Array [
  <h1>
    404.
    <br />
    <small>
      Page not found
    </small>
  </h1>,
  <a
    href="/"
    onClick={[Function]}
  >
    Go to main page
  </a>,
]
`;
